### ISSUE TITLE
Iron boots is inside Ganon's castle, need water medallion from water temple

### ISSUE THREAD

**Issue by Caronte64** [AmazingAmpharos/OoT-Randomizer#229]
A sheika stone told me that iron boots are inside Ganon's Castle. How do I enter the water temple then? I already have longshot and the longshot. I know there are some methods to enter the temple, but I though randomizer was made to be completed glitchless

**Comment by flagrama** [comment]
Do you need to beat the Water Temple? Post your spoiler log.

**Comment by rlbond86** [comment]
Water medallion is not necessarily in the water temple.

If Morpha gives you a spiritual stone you wouldn't need to go in at all.

**Comment by Caronte64** [comment]
My seed is 0W3ZNLZI92. The spoiler log was clear "Morpha: Water Medallion", also the altar from temple of time said the same.
Just to be clear, I already got that medallion, but I had to use a trick to open the gate. I simply wanted to tell about this, in case this is considered an error in the randomizer logic

**Comment by flagrama** [comment]
Please upload your spoiler by dragging it into the message box. We cannot properly verify the bug without the whole spoiler log.

**Comment by Caronte64** [comment]
[OoTR_MQ.txt](https://github.com/AmazingAmpharos/OoT-Randomizer/files/2773874/OoTR_MQ.txt)

**Comment by flagrama** [comment]
`shuffle_bosskeys:                          keysanity`
`logic_morpha_with_scale:                   True`

Pretty sure this means you were intended to enter Water Temple with the Gold Scale and beat Morpha.

**Comment by flagrama** [comment]
```
9: {
  Deku Tree MQ After Spinning Log Chest: Progressive Hookshot
  Queen Gohma: Spirit Medallion
  Jabu Jabus Belly MQ Second Room Upper Chest: Progressive Scale
  Kokiri Forest Storms Grotto Chest: Goron Tunic
  Forest Temple MQ First Chest: Small Key (Forest Temple)
}
10: {
  Underwater Bottle: Progressive Scale
  Forest Temple MQ Chest Behind Lobby: Small Key (Forest Temple)
  Forest Temple MQ NE Outdoors Lower Chest: Small Key (Forest Temple)
}
11: {
  Morpha: Water Medallion
  Forest Temple MQ Boss Key Chest: Small Key (Forest Temple)
}
```
Yup. Gold Scale is only in logic for two checks otherwise and neither one shows in your playthrough, while the Gold Scale is right before Morpha.

And more importantly your Iron Boots are after Morpha in the playthrough.

**Comment by Caronte64** [comment]
My issue was that you need to shoot a gem with the hookshot to enter the temple, but you need to perfome this trick if you don't have iron boots
https://www.youtube.com/watch?v=LOq7XA4g5IM
I only wanted to bring this, in case this is considered an error in logic

**Comment by rlbond86** [comment]
This is intended behavior if you enable the trick "Morpha with gold scale"

**Comment by flagrama** [comment]
Yeah that YouTube video is literally the exact trick that checkbox refers to.

**Comment by P3rf3ctXZer0** [comment]
That doesn't look like a glitch as so much basic understanding of slopes and physics.
